src/Card/CardImageCap.jsx

```diff
@@ -1,18 +1,45 @@
 import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import Skeleton from 'react-loading-skeleton';
 import CardContext from './CardContext';
 
 const CardImageCap = React.forwardRef(({
   src,
   srcAlt,
   logoSrc,
   logoAlt,
+  skeletonHeight,
+  skeletonWidth,
+  capLogoSkeleton,
+  capLogoSkeletonHeight,
+  capLogoSkeletonWidth,
```

Comment:
Not sure we need the `cap` prefix on these props. I think just `logoSkeleton`, `logoSkeletonHeight`, and `logoSkeletonWidth` could work just as well, since it's assumed were in the ImageCap component already?

Reply:
Yes, it's more simple variant